src/psfmachine/tpf.py

```diff
@@ -65,22 +65,15 @@ def __init__(
         )
         self.tpfs = tpfs
 
-        # Cut out 1.5 days after every data gap
-        dt = np.hstack([10, np.diff(time)])
-        focus_mask = ~np.in1d(
-            np.arange(len(time)),
-            np.hstack(
-                [
-                    np.arange(t, t + int(1.5 / np.median(dt)))
-                    for t in np.where(dt > (np.median(dt) * 5))[0]
-                ]
-            ),
-        )
-        if time_mask is None:
+        # combine focus mask and time mask if they exist
+        if time_mask is None and focus_mask is None:
+            self.time_mask = np.ones(len(self.time), bool)
+        elif time_mask is None and focus_mask is not None:
             self.time_mask = focus_mask
+        elif time_mask is not None and focus_mask is None:
+            self.time_mask = time_mask
```

Comment:
Here is where the `time_mask` and `focus_mask` are applied depending on the case.